### PR TITLE
#29: Refactored finding and deleting instances

### LINKED ISSUE #29: Support finding and deleting instances

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #29

### CODE REVIEW

File: doc/changes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Changes
 
+* [0.1.13](changes_0.1.13.md)
 * [0.1.12](changes_0.1.12.md)
 * [0.1.11](changes_0.1.11.md)
 * [0.1.10](changes_0.1.10.md)
```
Comment (ckunki) [inline doc/changes/changelog.md:6]: .gitattributes?

File: doc/changes/changes_0.1.13.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+This release contains the following changes:
+
+* SqlClient
+    * Method `runQuery` renamed to `execute`
+    * Added method `query` which returns the result of a `SELECT` query
+* Extension:
+    * Method `deleteInstance` removes parameters `installation: Installation, instance: Instance` and adds `instanceId: string`
+    * Method `readInstanceParameters` removes parameters `installation: Installation, instance: Instance` and adds `instanceId: string`
+    * Method `findInstances` removes parameter `installation: Installation` and adds `version: string`
```
Comment (ckunki) [inline doc/changes/changes_0.1.13.md:15]:
```suggestion
* SqlClient
    * Renamed Method `runQuery` to `execute`
    * Added method `query` which returns the result of a `SELECT` query
* Extension:
    * Method `deleteInstance`: removed parameters `installation: Installation, instance: Instance` and added `instanceId: string`
    * Method `readInstanceParameters`: removed parameters `installation: Installation, instance: Instance` and added `instanceId: string`
    * Method `findInstances`: removed parameter `installation: Installation` and added `version: string`
```

File: src/exasolSchema.ts [hunk 3]
```diff
@@ -31,27 +31,31 @@ export interface ExaAllScriptsRow {
...
+    adapterNotes: string
 }
 
 /**
  * The content of the SYS.EXA_ALL_VIRTUAL_SCHEMA_PROPERTIES table.
  */
-export interface ExaAllVirtualSchemaProperties {
-    rows: ExaAllVirtualSchemaPropertiesRow[]
+export interface ExaVirtualSchemaProperties {
+    rows: ExaVirtualSchemaPropertiesRow[]
```
Comment (ckunki) [inline src/exasolSchema.ts:53]: 👍

**Review by ckunki — CHANGES_REQUESTED** [review]
Some tiny text proposals, .gitattributes

File: doc/changes/changelog.md (hunk 1 above)
Comment (kaklakariada) [reply doc/changes/changelog.md:6]: No, pk does not support npm yet ;)

Comment (kaklakariada) [reply doc/changes/changelog.md:6]: See https://github.com/exasol/project-keeper/issues/373

**Review by ckunki — APPROVED** [review]

**Pull request merged**